dts/bindings/sensor/invensense,icm20948.yaml

```diff
@@ -0,0 +1,153 @@
+#  Copyright (c) 2023, Lazaro O'Farrill
+#  email: lazaroofarrill@gmail.com
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  ICM20948 Motion sensor by invensense
+
+compatible: "invensense,icm20948"
+
+include: [sensor-device.yaml, i2c-device.yaml]
+
+properties:
+  int-gpios:
+    type: phandle-array
+    description: |
+      The INT signal default configuration is active-high. The
+      property value should ensure the flags properly describe the
+      signal that is presented to the driver.
+
+  accel-fchoice:
+    type: int
+    description: |
+      Toggle accelerometer filter in low-power mode.
+      Maps to ACCEL_FCHOICE field in ACCEL_CONFIG.
+    default: 1
```

Comment:
The property description needs to explain why the default value was chosen. Is it a register reset value? Please update all properties in this binding that have default values.